### ISSUE TITLE
Cookies for sub only VOD

### ISSUE THREAD

**Issue by yovanoc** [HugoJF/twitch-tools#1]
I had an error today

<img width="1225" alt="image" src="https://user-images.githubusercontent.com/7302317/127472285-a325ab8d-6190-4c06-b494-97e7087ef38f.png">

I succeed to get it working like this while giving a netscape cookies file but it should be an option directly in the core I think. :)

```ts
youtubedl().getVideoInfo([this.videoUrl, "--cookies", "./twitch.tv_cookies.txt"]);
```

**Comment by HugoJF** [comment]
Was youtube-dl recently downloaded? Generally I expect them to fix these kind of errors.

**Comment by yovanoc** [comment]
Downloaded today. But its not really a bug, idk if there is a solution, when a VOD is for subscribers only on a twitch channel we need to give cookies to youtube-dl so idk how can this be properly handled but its not really a bug for me

**Comment by yovanoc** [comment]
For info, this is just the `auth-token` cookie that we need, so maybe you can ask for login/password of an account then generate this token or idk

**Comment by HugoJF** [comment]
Oooh I get it now. I'm gonna take a closer look once ai have some time.

I have an idea on how this could be fixed without interacting with the user (this project is just a backend for developers). 

Thanks for reporting this!

**Comment by yovanoc** [comment]
Yes I use this lib directly because I just need to download a single VOD.

Ahah no worries, good luck :)

**Comment by HugoJF** [comment]
Happy to hear that you found this useful even without any documentation hehe

I plan on updating twitch-clip-downloader to be more useful for basic use-cases (download a VOD, list of VODs, all VODs, a single clip, clip list, etc). So if you ever need something like that again, check the repo for updates.
